README.md

```diff
@@ -42,83 +34,109 @@ Run `source environment`  now and whenever these environment files are modified.
 #### Configuring cloud-specific access credentials
 
 ##### AWS
-* Follow the instructions in http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html to get the
-  `aws` command line utility.
-* Create an S3 bucket that you want DSS to use.
-* In `environment.local`, set the environment variable `DSS_S3_BUCKET_TEST`.
-* If you wish to run the unit tests, you must create a second S3 bucket to store the test fixtures, and set the
-  environment variable `DSS_S3_BUCKET_TEST_FIXTURES` to the name of that bucket.
+
+* Follow the instructions in http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html to get the `aws` command line utility.
+
+* Create an S3 bucket that you want DSS to use and in `environment.local`, set the environment variable `DSS_S3_BUCKET` to the name of that bucket. Make sure the bucket region is consistent with `AWS_DEFAULT_REGION` in `environment.local`.
```

Comment:
I don't know if it's confusing that we're telling people to set environment variables vs `environment.local`.

Reply:
It was clear to me when I went through it. It's a common pattern.